guides/doc-Administering_Red_Hat_Satellite/topics/proc_integrating-satellite-with-red-hat-single-sign-on-for-external-authentication.adoc

```diff
@@ -242,13 +252,14 @@ For more information about group mappers, see https://access.redhat.com/document
 --value "_{Keycloak-short}.example.com_/auth/realms/_{Keycloak-short}_Realm_/protocol/openid-connect/certs"
 ----
 +
-. Until https://bugzilla.redhat.com/show_bug.cgi?id=1792131[BZ#1792131] is resolved, you must use the Hammer CLI to set the organization and location. To set the organization and location, you must first retrieve the ID of the {Keycloak} authentication source:
+. To set the organization and location of the authentication source:
+.. First, retrieve the ID of the {Keycloak} authentication source.
 +
 ----
 # hammer auth-source external list
 ----
 +
-. Set the location and organization:
+.. Set the location and organization.
```

Comment:
```suggestion
.. Set the location and organization:
```
Throughout Satellite documentation, we use colons at the end of steps that are followed by a command.

Reply:
Ack, noted.